sqlalchemy_jsonapi/serializer.py

```diff
@@ -459,7 +459,6 @@ def _check_instance_relationships_for_delete(self, instance):
             check_permission(instance, rel_key, Permissions.EDIT)
 
             if rel.cascade.delete:
-                # pragma: no cover
                 if rel.direction == MANYTOONE:
                     related = getattr(instance, rel_key)
```

Comment:
Note I am still unable to hit this piece of code. I have tried for a bit with different model set up and I am unable to hit it.